geopandas/io/sql.py

```diff
@@ -65,6 +65,11 @@ def _df_to_geodf(df, geom_col="geom", crs=None):
     if geom_col not in df:
         raise ValueError("Query missing geometry column '{}'".format(geom_col))
 
+    if df.columns.to_list().count(geom_col) > 1:
+        raise ValueError(
+            f"Duplicate geometry column '{geom_col}' detected in SQL query output. Only one geometry column is allowed."
```

Comment:
Can you make sure this line is shorter than 88 characters? To make linter happy.